app/Importer/UserImporter.php

```diff
@@ -60,11 +60,12 @@ public function createUserIfNotExists(array $row)
         }
         // This needs to be applied after the update logic, otherwise we'll overwrite user passwords
         // Issue #5408
-        $this->item['password'] = $this->tempPassword;
+        $this->item['password'] = bcrypt($this->tempPassword);
```

Comment:
We should probably use the `Crypt::encrypt()` facade here, since we use that elsewhere, and down the line, may use different encryption outside of bcrypt.

Reply:
This has a problem. The Crypt::encrypt() facade uses OpenSSL and the AES-256-CBC cipher according to the docs (https://laravel.com/docs/5.7/encryption) and, while I'm not totally sure because I don't know how to make sense of this code, the Auth::attempt() facade is using another hashing algorithm, making authentication impossible after I implemented the Crypt::encrypt() to this password.